MANIFEST.in

```diff
@@ -1,23 +1,14 @@
-include COPYING.md
-include CONTRIBUTING.md
-include README.md
+include README.md LICENSE.md CONTRIBUTING.md requirements.txt requirements-test.txt requirements-doc.txt
 
 # Documentation
-graft docs
-exclude docs/\#*
-exclude docs/_*
-
-# Examples
-graft examples
-
-# docs subdirs we want to skip
-prune docs/build
-prune docs/gh-pages
-prune docs/dist
+exclude docs
 
 # Patterns to exclude from any directory
 global-exclude *~
+global-exclude __pycache__
 global-exclude *.pyc
 global-exclude *.pyo
+global-exclude *.orig
 global-exclude .git
+global-exclude .github
```

Comment:
I've found `.github` dir in the .tar.gz of the latest release. This is not something that should be included here.